sale_order_price_recalculation/README.rst

```diff
@@ -0,0 +1,63 @@
+===================================================
+Recalculation of sales order lines prices on demand
+===================================================
+
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```

Comment:
OK, let's move then with this module in any position you want, and wait for the answer and decide on maintainer-tools about the other topic, OK?